steps/telegram-notify/v1.0.0/step.yml

```diff
@@ -0,0 +1,59 @@
+title: telegram_notify
+summary: |
+  Notify group or chat about build status
+description: |
+  Notify group or chat about build status
+website: https://github.com/mcaldas/bitrise-step-telegram-notify
+source_code_url: https://github.com/mcaldas/bitrise-step-telegram-notify
+support_url: https://github.com/mcaldas/bitrise-step-telegram-notify/issues
+published_at: 2018-10-19T01:20:47.927152557+03:00
+source:
+  git: https://github.com/mcaldas/bitrise-step-telegram-notify.git
+  commit: b46ebf9c4bea48b5e2680aa8f5e848f476d05764
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
```

Comment:
For notification type steps, we set: `is_always_run: true`, to make sure the notifications are sent, even if the build failed (otherwise the step's execution will fail, if any previous step fails)